user-guide/Advanced_Functionality/DataMiner_Systems/Adding_a_DMA/Adding_a_regular_DataMiner_Agent.md

```diff
@@ -11,6 +11,7 @@ uid: Adding_a_regular_DataMiner_Agent
 > [!IMPORTANT]
 >
 > - Before you add a DataMiner Agent, make sure the IP network ports are configured correctly. See [Configuring the IP network ports](xref:Configuring_the_IP_network_ports).
+> - If the DataMiner System uses [Dedicated clustered storage](xref:Dedicated_clustered_storage), the new agent should be configured to use the same clustered storage in order to join the DMS.
```

Comment:
I'd like to add a bit more info so users also know how to configure this:
```suggestion
> - If the DataMiner System uses [Dedicated clustered storage](xref:Dedicated_clustered_storage), make sure the new DMA is configured to use the same clustered storage as the DMS you are about to add it to (by modifying the general database settings in its [DB.xml](xref:DB_xml) file to match those of the other DMAs and then restarting the DMA).
```

Reply:
I wanted to be as brief as possible, but if you believe more detail is necessary in this case, I agree with your suggestion.